src/components/navigation/header/index.stories.tsx

```diff
@@ -13,14 +13,40 @@ import {
 
 import Navigation from './index';
 
+const Wrapper: typeof Navigation = ({ user, selectTenant, ...props }) => {
```

Comment:
I added this wrapper to:
- actually change the selected tenant in storybook
- showcase the delay that will occur when we set the cookie and request the new token